File: src/DriverApp/app/src/main/java/com/abona_erp/driver/app/data/entity/TaskStatus.java
package com.abona_erp.driver.app.data.entity;

public class TaskStatus {

    private LastActivity data;
    private boolean expanded = false;

    public TaskStatus(LastActivity data) {
        this.data = data;
    }

    public void setExpanded(boolean expanded) {
        this.expanded = expanded;
    }

    public boolean isExpanded() {
        return expanded;
    }

    public LastActivity getData() {
        return data;
    }

    public void setData(LastActivity data) {
        this.data = data;
    }
}

File: app/src/main/java/com/abona_erp/driver/app/data/local/db/TaskStatus.kt
package com.abona_erp.driver.app.data.local.db
import com.google.gson.annotations.SerializedName

enum class TaskStatus (var status: Int){ //
    @SerializedName("0") PENDING(0),
    @SerializedName("50") RUNNING(50),
    @SerializedName("51") BREAK(51),
    @SerializedName("90") CMR(90),
    @SerializedName("100") FINISHED(100)
}

